scripts/conveyorBelt.js

```diff
@@ -0,0 +1,16 @@
+class ConveyorBelt extends Cube {
+  constructor(x, y, name, direction) {
+    super(x, y, AssetStore.getTexture(`conveyorBelt${direction}`));
+    this.direction = direction;
+  }
+  onCollision(collisionSide) {
+    if (collisionSide === "bottom") {
+      PLAYER_CUBE.stop(PLAYER_CUBE.x, PLAYER_CUBE.y - PLAYER_CUBE.velocityY);
+    } else if (collisionSide === "top") {
+      PLAYER_CUBE.stop(PLAYER_CUBE.x, PLAYER_CUBE.y - PLAYER_CUBE.velocityY);
```

Comment:
Should the behavior be the same for both bottom and top collision?

Reply:
Lol idk, I just copied this from the collision inside our Obstacle class x) This code is really old o.O so something is probably wrong. I'll look into it.

Reply:
Fixed.